openadapt/record.py

```diff
@@ -665,6 +707,10 @@ def record(
     except KeyboardInterrupt:
         terminate_event.set()
 
+    term_pipe_parent_window.send(window_write_q.qsize())
```

Comment:
File "/Users/aaron/Documents/GitHub/OpenAdapt/openadapt/record.py", line 710, in record
    term_pipe_parent_window.send(window_write_q.qsize())
  File "/opt/homebrew/Cellar/python@3.10/3.10.12/Frameworks/Python.framework/Versions/3.10/lib/python3.10/multiprocessing/queues.py", line 126, in qsize
    return self._maxsize - self._sem._semlock._get_value()
NotImplementedError

Reply:
Interesting I do not get that.

Can you investigate how could you resolve the error?